Code/Legacy/CrySystem/Log.cpp

```diff
@@ -838,7 +827,7 @@ void CLog::PopAssetScopeName()
 //////////////////////////////////////////////////////////////////////////
 const char* CLog::GetAssetScopeString()
 {
-    CryAutoCriticalSection scope_lock(m_assetScopeQueueLock);
+    AZStd::lock_guard<AZStd::mutex> scope_lock(m_assetScopeQueueLock);
```

Comment:
This can be changed to be

```suggestion
    AZStd::scoped_lock scope_lock(m_assetScopeQueueLock);
```